compute_sdk/globus_compute_sdk/sdk/client.py

```diff
@@ -750,6 +751,10 @@ def register_function(
             appropriate permissions
 
             DEPRECATED - ingesting functions to Globus Search is not currently supported
+        ha_endpoint_id : UUID | str | None
+            Users will only be able to run this function on the specified HA endpoint.
+            Since HA functions cannot be shared, this argument is mutually exclusive
+            with the ``group`` and ``public`` arguments.
```

Comment:
Word choice: I might use "As HA functions ..." rather than "Since " in this case.  "As" in this context would be slightly more formal in tone.

Reply:
I'm more of a "since" guy myself.

Reply:
Informally nice to meet you.